main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -580,17 +586,34 @@ private void setButtonInputValuesFromGP() {
     private boolean areCurrentCoordinatesValid() {
         // convert text to geopoint
         final Geopoint current;
-
         final String lat = getLatResult();
         final String lon = getLonResult();
 
         if (lat.contains("_") || lat.contains("*") || lon.contains("_") || lon.contains("*")) {
             return false;
         }
 
-        try {
-            current = new Geopoint(lat, lon);

```

Comment:
@S-Bartfast wasn't this just changed the other way around?

Reply:
Yeah, I've definitely managed to get something "arse about face" as we say in Australia.

Reply:
Reverted.